bank_statement_import_payment_fix/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module was written to extend the functionality of importing payments as bank statement lines.
+It invert values of "communication" and "reference" fields
+
+Installation
+============
+
+To install this module, you need to:
+
+* install it
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* nothing
+
+Usage
+=====
+
+To use this module, you need to:
+
+* nothing
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
```

Comment:
replace placeholders {project_repo}, {module_name}, {version} with the appropriate values